main.go

```diff
@@ -0,0 +1,11 @@
+/*
+Copyright © 2025 NAME HERE <EMAIL ADDRESS>
+
+*/
+package main
```

Comment:
I think we should actually move main.go into the project root so that the command can be installed with `go install github.com/fido-device-onboard/go-fdo-server@latest`.

Reply:
... Yes. GitHub's UI confused me.